packages/material-ui/src/TablePagination/TablePagination.test.js

```diff
@@ -200,29 +201,28 @@ describe('<TablePagination />', () => {
       assert.strictEqual(page, 0);
     });
 
-    it('should display 0 as start number if the table is empty ', () => {
```

Comment:
So this is a breaking change then? Otherwise please keep this test and add a new one.

Reply:
reverted and commited